sdk/schemaregistry/schema-registry-avro/src/schemaRegistryAvroSerializer.ts

```diff
@@ -202,11 +199,6 @@ export class SchemaRegistryAvroSerializer {
       id = (await this.registry.registerSchema(description)).id;
     } else {
       const response = await this.registry.getSchemaProperties(description);
-      if (!response) {
-        throw new Error(
-          `Schema '${description.name}' not found in registry group '${description.groupName}', or not found to have matching content.`
```

Comment:
I agree we need to remove these checks, but I wonder what the dev experience looks like now. 

Is the exception that gets bubbled up as easy to understand? Or might it be worth try/catching and throwing some error with this message and the original one as an innerError or something? (Feel free to file an issue for this instead of handling it in this PR)